ui/cockpit/client/scripts/directives/change-version.js

```diff
@@ -0,0 +1,166 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+var fs = require('fs');
+var angular = require('angular');
+
+var template = fs.readFileSync(__dirname + '/change-version.html', 'utf8');
+
+module.exports = [
+  '$timeout',
+  '$location',
+  'ProcessDefinitionResource',
+  'camAPI',
+  'Notifications',
+  '$translate',
+  function(
+    $timeout,
+    $location,
+    ProcessDefinitionResource,
+    camAPI,
+    Notifications,
+    $translate
+  ) {
+    return {
+      restrict: 'A',
+      template: template,
+      scope: {
+        definition: '=changeVersion',
+        type: '@',
+        history: '@?'
+      },
+      link: function($scope) {
+        $scope.model = {};
+        $scope.model.newVersion = $scope.definition.version;
+
+        $scope.isValid = false;
+        $scope.isValidating = true;
+
+        $scope.isActive = false;
+
+        $scope.enableValidating = function() {
+          $scope.isValidating = true;
+        };
+
+        $scope.changeLocation = function() {
+          $timeout(function() {
+            var path = '/';
+            if ($scope.type === 'drd') {
+              path += 'decision-requirement/';
+            } else {
+              path += $scope.type + '-definition/';
+            }
+
+            path += $scope.newDefinition;
+            if ($scope.history !== undefined) {
+              path += '/history';
+            }
+
+            $location.path(path);
+          });
+        };
+
+        $scope.change = function(form) {
+          $scope.isValid = form.$valid;
+
+          if ($scope.model.newVersion == $scope.storedVersion) {
```

Comment:
Lets also validate that the Version is `>0` before making a request

```suggestion
          if ($scope.model.newVersion == $scope.storedVersion || $scope.model.newVersion < 1) {
```